### PR TITLE
publish Paleta 0.1.0

### PR DESCRIPTION
**Pull request by nate-xyz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. 
- [x] All assets referenced in the manifest are redistributable by any party. 
- [x] I am an upstream contributor to the project. 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.nate_xyz.Paleta.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "app-id" : "com.github.nate_xyz.Paleta",
```
Comment (hfiguiere) [inline com.github.nate_xyz.Paleta.json:2]: it should be `io.github`

File: com.github.nate_xyz.Paleta.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "pypi-dependencies.json",
+        {
+            "name" : "paleta",
+            "builddir" : true,
```
Comment (hfiguiere) [inline com.github.nate_xyz.Paleta.json:29]:
```suggestion
```

this is on by default with meson

File: pypi-dependencies.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/56/18/be03b7058e65f9df479b14e7af4e73945ce311e07aaad45cf2536e14791a/colorthief-0.2.1-py2.py3-none-any.whl",
+                    "sha256": "b04fc8ce5cf9c888768745e29cb19b7b688d5711af6fba26e8057debabec56b9"
+                }
+            ]
+        },
+        {
+            "name": "python3-Pillow",
```
Comment (hfiguiere) [inline pypi-dependencies.json:26]: you probably don't need to add this one as it is just above

**Comment by hfiguiere** [comment]
bot, build io.github.nate_xyz.Paleta

**Comment by nate-xyz** [comment]
bot, build io.github.nate_xyz.Paleta

**Pull request closed without merging**